src/sidebar/components/mentions/MentionPopoverContent.tsx

```diff
@@ -21,15 +23,25 @@ export default function MentionPopoverContent({
   }
 
   return (
-    <div className="flex flex-col gap-y-1.5">
-      <div data-testid="username" className="text-md font-bold">
-        @{content.username}
+    <div className="flex flex-col gap-y-4">
+      <div className="flex flex-col gap-y-1.5">
+        <div data-testid="username" className="text-md font-bold">
+          @{content.username}
+        </div>
+        {content.display_name && (
+          <div data-testid="display-name" className="text-color-text-light">
+            {content.display_name}
+          </div>
+        )}
       </div>
-      {content.display_name && (
-        <div data-testid="display-name" className="text-color-text-light">
-          {content.display_name}
+      {content.description && (
+        <div data-testid="description" className="line-clamp-2">
+          {content.description}
         </div>
       )}
+      <div data-testid="created" className="text-color-text-light">
+        Joined in <b>{formatDateTime(content.created)}</b>
```

Comment:
In English this would need to be "Joined _on_ {date}". You can make this work across multiple languages by just omitting the preposition entirely. An example from Discourse:

<img width="663" alt="discourse-joined-date-2" src="https://github.com/user-attachments/assets/1aa90241-5c41-4d05-b6f3-ee06c15633d3" />

I would also omit the _time_ here and just keep the date. It takes up space and I can't see that it adds value. The date could perhaps also be simplified to just month and year, but that's something we can discuss.